spec/src/main/asciidoc/changelog.adoc

```diff
@@ -66,12 +66,14 @@
 *** Added `Timer.getElapsedTime()` which returns `java.time.Duration`
 ** Removed `MetadataBuilder.withOptional*` methods
 ** Global tags and `_app` tag are no longer handled automatically by the `MetricID` class, the implementation is expected to add them by itself, for example during metric export
+** Added the `Histogram.getSum()` which returns `long` (https://github.com/eclipse/microprofile-metrics/issues/597[#597)
 
 === Functional Changes
 ** Simple Timer metrics now track the highest and lowest recorded timing duration of the previous completed minute (https://github.com/eclipse/microprofile-metrics/issues/523[#523])
 ** Timer now exposes total elapsed time duration as a metric value (https://github.com/eclipse/microprofile-metrics/issues/524[#524])
 ** Clarified that the existing REST metric `REST.request` will not monitor and track a REST request to a REST endpoint if an unmapped exception occurs.
 ** Introduced a new base REST metric `REST.request.unmappedException.total` that counts the occurrences of unmapped exceptions for each REST endpoint (https://github.com/eclipse/microprofile-metrics/issues/533[#533])
+** Histogram now exposes the total sum of recorded values (https://github.com/eclipse/microprofile-metrics/issues/597[#597)
```

Comment:
** Histogram now exposes the total sum of recorded values (https://github.com/eclipse/microprofile-metrics/issues/597[#597])